src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -12,6 +12,13 @@
 short-summary: Manage Azure Network resources.
 """
 
+helps['network list-service-tags'] = """
+type: command
+long-summary: >
```

Comment:
What does `>` mean here?

Reply:
you can split the long summary into multiple lines